tests/app/organisation/test_rest.py

```diff
@@ -37,3 +37,52 @@ def test_get_organisation_by_id(notify_api, notify_db, notify_db_session):
     assert organisation['logo'] == '/path/image.png'
     assert organisation['name'] == 'My Org'
     assert organisation['id'] == str(org.id)
+
+
+def test_create_organisation(client, notify_db, notify_db_session):
+    data = {
+        'name': 'test organisation',
+        'colour': '#0000ff',
+        'logo': '/images/test_x2.png'
+    }
+    auth_header = create_authorization_header()
+
+    response = client.post(
+        '/organisation',
+        headers=[('Content-Type', 'application/json'), auth_header],
+        data=json.dumps(data)
+    )
+    assert response.status_code == 201
+    json_resp = json.loads(response.get_data(as_text=True))
+    assert data['name'] == json_resp['data']['name']
+
+
+def test_create_organisation_without_logo_raises_error(client, notify_db, notify_db_session):
+    data = {
+        'name': 'test organisation',
+        'colour': '#0000ff',
+    }
+    auth_header = create_authorization_header()
+
+    response = client.post(
```

Comment:
This could be cleaner if you used the `admin_request` fixture that Leo made. For example:

```python
response = admin_request.post(
  '/organisation',
  data={
    'name': 'test organisation',
    'colour': '#0000ff',
  }
  _expected_status=400
)
  
assert response == {…}
```